shared/rakefiles/test.rake

```diff
@@ -11,11 +11,27 @@ task :test do
   exit locust_status unless locust_status == 0
 end
 
-desc "[TEST] Run Locust swarm against Preferences service in current cluster"
-task :test_preferences => [:set_vars, :check_destroy_allowed] do
+desc "[TEST] Run Locust swarm against Preferences service (READ) in current cluster"
+task :test_preferences_read => [:set_vars, :check_destroy_allowed] do
   ENV["TF_VAR_locust_target_host"] = "http://preferences.gpii.svc.cluster.local"
   ENV["TF_VAR_locust_target_app"] = "preferences"
-  ENV["TF_VAR_locust_script"] = "preferences.py"
+  ENV["TF_VAR_locust_stackdriver_label"] = "preferences"
+  ENV["TF_VAR_locust_script"] = "preferences_read.py"
+  ENV["TF_VAR_locust_desired_median_response_time"] = "300"
+  ENV["TF_VAR_locust_desired_max_response_time"] = "1500"
+
+  Rake::Task[:set_compose_env].reenable
+  Rake::Task[:set_compose_env].invoke
+  Rake::Task[:test].reenable
+  Rake::Task[:test].invoke
+end
+
+desc "[TEST] Run Locust swarm against Preferences service (WRITE) in current cluster"
+task :test_preferences_write => [:set_vars, :check_destroy_allowed] do
+  ENV["TF_VAR_locust_target_host"] = "http://preferences.gpii.svc.cluster.local"
+  ENV["TF_VAR_locust_target_app"] = "preferences"
```

Comment:
In case this variable stays the same for both tests – there would be no way for us to differentiate results in Stackdriver. Check https://github.com/gpii-ops/gpii-infra/blob/master/gcp/modules/locust/swarm.tf#L98.

Reply:
@natarajaya, once I realised that the write tests were being saved to the same place, I changed the label for the "write" tests.  I left the label for the "read" tests alone, but can easily change that to "preferences_read" if you like.

Reply:
@the-t-in-rtf I am okay with using new env variable, but the code I am referring to in my previous comment, still uses `locust_target_app` as metric name for [Stackdriver client](https://github.com/gpii-ops/gpii-infra/blob/master/gcp/modules/locust/client.rb#L40-L55).

Reply:
So, can I just eliminate the new variable and change `locust_target_app` for the write tests?

Reply:
Turns out I can, and did.